storage/disk/paths.go

```diff
@@ -66,7 +68,7 @@ func (ps pathSet) IsDisjoint() bool {
 	for i := range ps {
 		for j := range ps {
 			if i != j {
-				if ps[i].HasPrefix(ps[j]) || ps[j].HasPrefix(ps[i]) {
+				if hasPrefixWithWildcard(ps[i], ps[j]) {
```

Comment:
🤦 good catch--we don't have to compare both ways because we're looping over all pairs.